packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettings.tsx

```diff
@@ -61,9 +75,25 @@ export const DatabaseSettings: React.FC<Props> = (props) => {
     setIsResetRootPasswordDialogOpen(false);
   };
 
+  const onSuspendDialogClose = () => {
+    setIsSuspendClusterDialogOpen(false);
+  };
+
   return (
     <>
       <Paper>
+        {isDatabasesV2GA && isDefault ? (
+          <>
+            <DatabaseSettingsMenuItem
+              buttonText={'Suspend Cluster'}
+              descriptiveText={suspendClusterCopy}
+              disabled={disabled || database.status !== 'active'}
+              onClick={onSuspendCluster}
+              sectionTitle={'Suspend Cluster'}
+            />
+            <Divider spacingBottom={22} spacingTop={28} />
+          </>
+        ) : null}
```

Comment:
This is fine, but don't hesitate to use the `&&` notation. It should be fine in situations like this. I find that it makes the JSX a bit cleaner 🧼 

```suggestion
        {isDatabasesV2GA && isDefault && (
          <>
            <DatabaseSettingsMenuItem
              buttonText={'Suspend Cluster'}
              descriptiveText={suspendClusterCopy}
              disabled={disabled || database.status !== 'active'}
              onClick={onSuspendCluster}
              sectionTitle={'Suspend Cluster'}
            />
            <Divider spacingBottom={22} spacingTop={28} />
          </>
        )}
```